sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Primitives/ServiceBusMessageActions.cs

```diff
@@ -34,6 +34,17 @@ internal ServiceBusMessageActions(ServiceBusReceiver receiver)
             _receiver = receiver;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServiceBusMessageActions"/> class for mocking use in testing.
+        /// </summary>
+        /// <remarks>
+        /// This constructor exists only to support mocking. When used, class state is not fully initialized, and
+        /// will not function correctly; virtual members are meant to be mocked.
+        ///</remarks>
+        protected ServiceBusMessageActions()
```

Comment:
I think we will need to add this for the derived class too - ServiceBusSesssionMessageActions.

Reply:
My bad;  didn't see it when I patched this.   I'll push that out in another PR.